sxiv.h

```diff
@@ -407,8 +407,10 @@ struct win {
 	Window xwin;
 	win_env_t env;
 
-	XftColor bg;
-	XftColor fg;
+	XftColor bg_color;
+	XftColor fg_color;
+	XftColor bar_color;
+	XftColor text_color;
```

Comment:
Lets keep `bg` and `fg` as is and name the bar colors `bar_fg` and `bar_bg`.

Reply:
Idk, what the other people think about that. For me is makes the code more readable this way, and its not that big of a change.

Reply:
Since the scope of this pr is to add custom bar colors, we should leave anything else untouched.
If other's also think new names are more readable, then it should be it's own commit imo.

Reply:
@GRFreire are you fine with `win_bg`, `bar_bg` as var names? This mirrors the Xresources entries so I think it'd be more appropriate.

Reply:
all left is to resolve this and the pr can be merged.

Reply:
Yeah, changing to `win_bg/fg` and `bar_bg/fb` then

Reply:
I think we're also changing the default bar color values.

Reply:
Yes, changed that on my last commit